aspnet/tutorials/publish-to-azure-webapp-using-vs.rst

```diff
@@ -1,47 +1,181 @@
+
+The app displays the email registration used to register and a **Log off** link.
+
+
+.. image:: publish-to-azure-webapp-using-vs/_static/hello.png
+
+Deploy the app to Azure
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Right click on the project and select **Publish**
+
+.. image:: publish-to-azure-webapp-using-vs/_static/pub.png
+
+In the **Publish** dialog, tap **Microsoft Azure App Service**
+
+.. image:: publish-to-azure-webapp-using-vs/_static/maas1.png
+
+Tap **New** to enter a new resource group. Creating a new resource group will make it easy to delete all the Azure resources you create in this tutorial.
+
+.. image:: publish-to-azure-webapp-using-vs/_static/newrg1.png
+
+Create a new resource group and app service plan:
+
+- Tap **New** for the resource group and enter a name for the new resource group
+- Tap **New** for the  app service plan and select a location near you. You can keep the default generated name
+- Tap **Explore additional Azure services** to create a new database
+
+.. image:: publish-to-azure-webapp-using-vs/_static/cas.png
+
+- Tap the green **+** icon to create a new SQL Database
+
+.. image:: publish-to-azure-webapp-using-vs/_static/sql.png
+
+- Tap  **New** on the **Configure SQL Database** dialog
+
+.. image:: publish-to-azure-webapp-using-vs/_static/conf.png
+
+- You can keep the default **Server Name**
+- Enter an administrator user name and password, and then tap **OK**
+
+.. image:: publish-to-azure-webapp-using-vs/_static/dbrick.png
+
+- Tap **OK** on the  **Configure SQL Database** dialog
+
+- Tap **Create** on the **Create App Service** dialog
+
+.. image:: publish-to-azure-webapp-using-vs/_static/create_as.png
+
+- Tap **Next** in the **Publish** dialog
+
+.. image:: publish-to-azure-webapp-using-vs/_static/pubc.png
+
+- On the **Settings** stage of the **Publish** dialog:
+
+  - Check **Use this connection string at runtime**
+  - Check **Apply this migration on publish**
+
+  .. image:: publish-to-azure-webapp-using-vs/_static/pubs.png
+
+- Tap **Publish**
+
+Visual Studio will publish your app to Azure and launch the cloud app in your browser.
+
+Test your app in Azure
+-----------------------
+
+- Test the **About** and **Contact** links
+- Register a new user
+
+.. image:: publish-to-azure-webapp-using-vs/_static/final.png
+
+Update the app
+--------------------
+
+- Edit the *Views/Home/About.cshtml* Razor view file. Change the content, for example:
+
+.. code-block:: html
+  :emphasize-lines: 7
+
+  @{
+      ViewData["Title"] = "About";
+  }
+  <h2>@ViewData["Title"].</h2>
+  <h3>@ViewData["Message"]</h3>
+
+  <p>My updated about page.</p>
+
+- Right click on the project and select **Publish**
+
+.. image:: publish-to-azure-webapp-using-vs/_static/pub.png
+
+- Tap **Publish** and verify the changes you made are displayed on Azure
+
+Clean up
+--------------
+
+When you have finished testing the app, log into the `Azure portal <https://portal.azure.com/>`__ and delete the app.
```

Comment:
You can also mention that they can delete the resource group as well to tie it to your point above about deleting everything.